docs/topics/strings.md

```diff
@@ -132,4 +141,36 @@ use the following syntax:
 val price = """
 ${'$'}_9.99
 """
-```
\ No newline at end of file
+```
+
+## String formatting
+To deal with complex formatting requirements, use the [`String.format()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/format.html) 
```

Comment:
It's important to note that unlike other features mentioned on this page, `String.format` is provided only in Kotlin/JVM

Reply:
That's correct. Added it.

Reply:
Done.